src/test/resources/com/znsio/teswiz/features/jioCinema.feature

```diff
@@ -9,4 +9,13 @@ Feature: jio cinema test
     When I swipe left on tending in india section
     Then I am able to view number 3 trending movie
     When I swipe right on tending in india section
-    Then I am able to view number 1 trending movie
\ No newline at end of file
+    Then I am able to view number 1 trending movie
+
+ #  CONFIG=./configs/jiocinema_local_config.properties PLATFORM=android TAG=horizontalSwipeUsingGesturePluginTest ./gradlew run
+  @android @swipe @horizontalSwipeUsingGesturePluginTest
+  Scenario: validating swipe functionality
+    Given I navigate to jio Cinema application's home page
+    When I swipe "left" trending no 2 on trending in india section
+    Then I am able to view number 3 trending movie
+    When I swipe "right" trending no 2 on trending in india section
+    Then I am able to view number 1 trending movie
```

Comment:
Remove 1 in trending movies

Reply:
the numbers in the step is the trending number, the action and validations are done with respect to trending number